charts/streams-app/templates/deployment.yaml

```diff
@@ -161,3 +168,13 @@ spec:
                 path: log4j.properties
         {{- end }}
       {{- end }}
+{{- range $key, $value := .Values.secrets }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ $key | lower}}"
+type: Opaque
+stringData:
+  value: {{ $value }}
```

Comment:
quote?

Reply:
`{{ $value | quote}}` or like `"{{ $value }}"`?

Reply:
Whatever we did elsewhere

Reply:
Apparently we use both in different places. For `name:` normally `"{{ $name }}"` and for `value:` usually `{{ $value | quote }}`. In the end both produces the same result.